packages/core-mobile/app/utils/js/retry.test.ts

```diff
@@ -68,4 +68,17 @@ describe('failure case', () => {
 
     expect(getAndIncrementCounter).toHaveBeenCalledTimes(2)
   })
+
+  it('should stop polling when isStopping is true', async () => {
```

Comment:
```suggestion
  it('should stop retrying when isStopping is true', async () => {
```
nit: let's keep the wording consistent.